docs/guide/packages/lucide-svelte.md

```diff
@@ -71,6 +81,87 @@ To customize the appearance of an icon, you can pass custom properties as props
 
 This results a filled phone icon.
 
+## Types
+
+The package includes type definitions for all icons. This is useful if you want to dynamically load icons with the `svelte:component` directive whether you are using TypeScript or JSDoc.
+
+### TypeScript Example
+
+```svelte
+<script lang="ts">
+  import { Home, Library, Cog } from 'lucide-svelte';
+  import type { ComponentType } from 'svelte';
+  import type { Icon } from 'lucide-svelte';
+
+  type Route = {
+    name: string;
+    href: string;
+    icon: ComponentType<Icon>;
+  }
+
+  const routes: Route[] = [
```

Comment:
For clearance nice to rename this to `menuItems`, I think that fits a bit better with what's happening in the component.

Reply:
Done, @ericfennis 

I've also made a minor update about the manner of import to use the recommended/fast way.